app/src/main/java/com/example/marvelapp/activity/CharactersActivity.kt

```diff
@@ -60,7 +59,7 @@ class CharactersActivity : AppCompatActivity(), KoinComponent, CharacterListener
     private fun displayFragment(id: String) {
         val characterFragment = CharacterFragment.newInstance(id)
         val fragmentManager: FragmentManager = supportFragmentManager
-        characterFragment.show(fragmentManager, Constants.TAG)
+        characterFragment.show(fragmentManager, TAG)
```

Comment:
this TAG constant should be in a companion object inside the Fragment

Reply:
done